src/components/Global/IframeWrapper/index.tsx

```diff
@@ -28,11 +29,28 @@ const IframeWrapper = ({
         }
     }, [onClose])
 
+    const areYouSurePromptRow = (
+        <div className='flex flex-col text-center sm:text-left sm:flex-row justify-between sm:items-center gap-4 p-2 sm:p-0 w-full'>
+            <p className="text-sm ml-1">Are you sure ? your KYC progress will be lost.</p>
```

Comment:
issue(blocking): message is specific to KYC flow, component is used in other flows

In particular, this message is confusing (and wrong) to get on the accept TOS flow (that must be closed manually also).
![2024-10-30_09-15](https://github.com/user-attachments/assets/b04f0c14-3290-4fdc-8668-a64e34f49c9c)


suggestion: I would add a confirmationMessage optional prop and show that message if exists or just direclty close if not

Reply:
Yes, component is only used in TOS & KYC but I agree

Reply:
I would rather not have confirmation on the TOS, because as part of normal flow the user must close the modal after acepting, and there is no progress to be lost (either they have accepted or not, is not too inconvinient to the user to close like in the kwy case). For example, here I completed and now want to close the modal but I'm afraid to "lose progress"
![2024-10-30_11-06](https://github.com/user-attachments/assets/f0ba779c-db89-4496-b192-2a2c4e80e53f)